quantum/action_tapping.c

```diff
@@ -186,25 +186,17 @@ bool process_tapping(keyrecord_t *keyp) {
                                     false
 #            if defined(HOLD_ON_OTHER_KEY_PRESS) || defined(HOLD_ON_OTHER_KEY_PRESS_PER_KEY)
                                     || (
-                                        IS_LT(tapping_keycode)
+                                        true
 #                ifdef HOLD_ON_OTHER_KEY_PRESS_PER_KEY
                                         && get_hold_on_other_key_press(tapping_keycode, keyp)
 #                endif
                                     )
-#            endif
-#            if !defined(IGNORE_MOD_TAP_INTERRUPT) || defined(IGNORE_MOD_TAP_INTERRUPT_PER_KEY)
-                                    || (
-                                        IS_MT(tapping_keycode)
-#                ifdef IGNORE_MOD_TAP_INTERRUPT_PER_KEY
-                                        && !get_ignore_mod_tap_interrupt(tapping_keycode, keyp)
-#                endif
-                                    )
 #            endif
                                 ) && tapping_key.tap.interrupted == true
```

Comment:
This should be moved to the other side of the parentheses for readability

Reply:
Do you mean this?:
```diff
diff --git a/quantum/action_tapping.c b/quantum/action_tapping.c
index 8ab9026937..1ca4bcf5c4 100644
--- a/quantum/action_tapping.c
+++ b/quantum/action_tapping.c
@@ -182,6 +182,7 @@ bool process_tapping(keyrecord_t *keyp) {
                         (
                             // Rolled over the two keys.
                             (
+                                tapping_key.tap.interrupted == true &&
                                 (
                                     false
 #            if defined(HOLD_ON_OTHER_KEY_PRESS) || defined(HOLD_ON_OTHER_KEY_PRESS_PER_KEY)
@@ -192,7 +193,7 @@ bool process_tapping(keyrecord_t *keyp) {
 #                endif
                                     )
 #            endif
-                                ) && tapping_key.tap.interrupted == true
+                                )
                             )
                             // Makes Retro Shift ignore the default behaviour
                             // of dual-role keys (MT and LT) for nested taps

```
You'd lose out on the early exit in the evaluation of the AND (since `tapping_key.interrupted == true` is now evaluated before `false`) and it doesn't seem that much more readable to me tbh

Reply:
The "rolled over the two keys" comment just makes it seem like that's already been determined to me and it took me a hot sec to find where that is actually checked. The loss of the early exit would get compiled out without `HOLD_ON_OTHER_KEY_PRESS` anyway.

Reply:
Solved in https://github.com/qmk/qmk_firmware/pull/15741/commits/c8eff5f911fa973a5be623589aa47d5fde854206